lib/diagnostic.rb

```diff
@@ -69,18 +69,18 @@
 
 # What value will be returned. Write your response as a Ruby string.
 
-Diagnostic.flow_control = 'your response here'
+Diagnostic.flow_control = 'The Dark Knight'
 
 # In a Ruby string, write what keyword you use for "else if" clauses in Ruby.
 
-Diagnostic.else_if = 'your response here'
+Diagnostic.else_if = 'elsif'
 
 # Returns in Ruby
 #
 # Does ruby require an explicit return from methods? Answer true or false, and
 # replace nil.
 
-Diagnostic.returns = nil
+Diagnostic.returns = true
```

Comment:
Ruby doesn't require an explicit `return`.  The most recently evaluated expression is the return value.  e.g.,

```ruby
def no_explicit_return
  1 + 1
end

p no_explicit_return # => 2
```